engine/src/flutter/impeller/entity/contents/content_context.cc

```diff
@@ -548,7 +549,7 @@ ContentContext::ContentContext(
 #endif  // IMPELLER_ENABLE_OPENGLES
 
   is_valid_ = true;
-  InitializeCommonlyUsedShadersIfNeeded();
+  // InitializeCommonlyUsedShadersIfNeeded();
```

Comment:
should be gaurded by lazy shader mode

Reply:
I was going to ask you about this.  This is how it is in the original PR.

Reply:
mistake on my part :)